package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "floeproject.org",
+  "version": "1.0.0",
+  "description": "FLOE Project Website",
+  "main": "index.js",
+  "scripts": {
+    "build": "eleventy",
+    "eleventy": "eleventy --serve",
+    "eleventyport": "eleventy --serve --port=9778"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fluid-project/floeproject.org.git"
+  },
+  "keywords": [
+    "Floe",
+    "Project",
+    "Flexible",
+    "Learning",
+    "for",
+    "Open",
+    "Education"
+  ],
+  "author": "OCAD University <idrc@ocadu.ca>",
+  "contributors": [
+    "Sachin Chopra <schopra@mt.iitr.ac.in> (www.github.com/sachin10101998)"
+  ],
+  "license": "ISC",
```

Comment:
The license should be [Creative Commons BY 4.0](https://creativecommons.org/licenses/by/4.0/). Here's an example of a [license file](https://github.com/fluid-project/.github/blob/master/LICENSE.md
) that should also be included in the repository.